web-ts/src/components/Layout.tsx

```diff
@@ -69,10 +70,9 @@ const Main = styled.main`
   }
 `
 
-// TODO IGAPP-646
-// const Aside = withPlatform(styled.aside<{ platform: Platform }>`
-//   position: ${props => (props.platform.positionStickyDisabled ? 'static' : 'sticky')};
```

Comment:
Could not reproduce this with any valid typing.

Reply:
I like it better this way anyway. Maybe even just pass `positionStickyDisabled` instead of the whole `platform`.